api/base/middleware.py

```diff
@@ -198,6 +199,15 @@ def process_response(self, request, response):
                         self.set_sloan_tags(user, sloan_flag_name, active)
                         self.set_sloan_cookie(f'dwf_{sloan_flag_name}', active, request, response)
 
+                        if provider.domain:
+                            self.set_sloan_cookie(
+                                f'dwf_{sloan_flag_name}_custom_domain',
```

Comment:
What if the person visits multiple custom domains? Is this still going to work, or are we going to need a different cookie name for each custom domain?

Reply:
cookies with different domains can have the same name, I'm thinking of possibilities where this could create a negative interaction, but can't. ES records provider data, so we will know where the view/download is coming from.

Reply:
Alternately, if there's not a namespace collision in that case, would it would just to keep it 'dwf_{sloan_flag_name}' with a different domain?

Reply:
Ha, actually because you can't SET two cookies with the same name in a single response, (not without overriding a lot of Django anyway).

Reply:
Okay, so it's just about the timing more than a namespace collision proper. 👍